aiida/backends/tests/work/test_process_function.py

```diff
@@ -29,64 +29,57 @@
 CUSTOM_DESCRIPTION = 'Custom description'
 
 
-class ProcessFunctionNode(FunctionCalculationMixin, ProcessNode):
-    """A dummy ORM node class to test base process functions."""
-    pass
-
-
 class TestProcessFunction(AiidaTestCase):
     """
-    Note that the @process_function decorator should never be used in production, which is why it does not have a node
-    class by default and we create a dummy class in this file especially for testing purposes. We just use it here to
-    test the generic interface functionality, that is shared between the concrete @calcfunction and @workfunction
-    variants. For that reason we also do not test the outputs here, because they will not be generated for the base
-    ProcessNode class and are specific to the @calculation and @workfunction, and will therefore be tested for them
-    separately.
+    Note that here we use @workfunction (and occasionally @calcfunction) because we test only the generic interface
```

Comment:
Suggestion:
```Note that here we use `@workfunctions` and `@calculations`, the concrete versions of the `@process_function` decorator, even though we are testing only the shared functionality that is captured in the `@process_function` decorator, relating to the transformation of the wrapped function into a `FunctionProcess`. The reason we do not use the `@process_function` decorator itself, is because it does not have a node class by default. We could create one on the fly, but then anytime inputs or outputs would be attached to it in the tests, the `validate_link` function would complain as the dummy node class is not recognized as a valid process node.```